svo (install the zip, not me).xml

```diff
@@ -54122,6 +54122,11 @@ function svo.gettreeableaffs()
   return table.n_intersection(svo.keystolist(svo.affl), svo.treecurables)
 end
 
+-- returns a list of affs that fous can get
```

Comment:
That'd be great. And while we are at documentation and comments... Could you add the new function to svof/doc/index.rst as well?